scripts/testpop/testpop.go

```diff
@@ -35,8 +35,13 @@ import (
 )
 
 var (
-	hostport        = flag.String("listen", "127.0.0.1:3000", "hostport to start ringpop on")
-	hostfile        = flag.String("hosts", "./hosts.json", "path to hosts file")
+	hostport = flag.String("listen", "127.0.0.1:3000", "hostport to start ringpop on")
+	hostfile = flag.String("hosts", "./hosts.json", "path to hosts file")
+
+	suspectPeriod   = flag.Int("suspect", 5, "suspect period in seconds")
```

Comment:
I'll add the "-period" prefix, I can make it ms. Didn't use go's duration flag because of node..